rasa/core/agent.py

```diff
@@ -519,6 +520,21 @@ def predict_next(self, sender_id: Text, **kwargs: Any) -> Optional[Dict[Text, An
             sender_id, action, output_channel, self.nlg, policy, confidence
         )
 
+    async def inject_intent(
```

Comment:
how about calling it (as we do it for the `MappingPolicy`)
```suggestion
    async def trigger_intent(
```